types/testingutils/network.go

```diff
@@ -7,11 +7,13 @@ import (
 
 type TestingNetwork struct {
 	BroadcastedMsgs []*types.SSVMessage
+	DKGOutputs      map[types.OperatorID]*dkg.SignedOutput
 }
 
 func NewTestingNetwork() *TestingNetwork {
 	return &TestingNetwork{
 		BroadcastedMsgs: make([]*types.SSVMessage, 0),
+		DKGOutputs:      make(map[types.OperatorID]*dkg.SignedOutput, 0),
```

Comment:
SSVMessages include in them DKG messages, the test should decode it from the SSVMessage like other tests are doing

Reply:
This is to align with `StreamDKGOutput` function we have. It's actually different from the network channel.
All the messages containing `SignedOutput` are actually received from the `network` then once all `SignedOutput` messages are received, these messages will be streamed to any local consumers. This is to cater to the second step.

Reply:
Will need to refactor later